src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -496,6 +502,11 @@ private String postRequest(PostMethod httppost) throws UnsupportedEncodingExcept
 
     }
 
+    private <T> String serialize(T o) throws IOException {
```

Comment:
This class must be refactored to follow the pattern of [BitbucketServerAPIClient](https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/a02519c7cfbde64cd309ccb8e7b08ebbdf0efed9/src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java), but this is out-of-scope in this PR.

Meanwhile, can you call this method `parse`? Just to be coherent with [BitbucketServerAPIClient.parse](https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/a02519c7cfbde64cd309ccb8e7b08ebbdf0efed9/src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java#L365).